apps/mini-app/src/routes/Search/Search copy.tsx

```diff
@@ -0,0 +1,32 @@
+import type { UserProfile } from '@samovar/models';
+import type { AxiosResponse } from 'axios';
+import { useQuery } from 'react-query';
+import { authenticatedHttpClient } from '../../services/http-client';
+import { SearchResultProfileItem } from '../../features/SearchResultList';
+
+function GetUserProfilesByUserId(): Promise<AxiosResponse<UserProfile[]>> {
+  return authenticatedHttpClient.get('api/v1/user/profiles');
+}
+
+export function Component() {
+  const result = useQuery('profiles', GetUserProfilesByUserId, {
+    enabled: Boolean(window.Telegram.WebApp.initData),
+  });
+  console.log('profiles result', result);
+  return (
+    <main>
+      {result.isLoading ? (
```

Comment:
fyi: useQuery has optional `suspense` param. Not important at this time.

Reply:
Noted